hq_superset/utils.py

```diff
@@ -0,0 +1,38 @@
+import sqlalchemy
+
+def get_datasource_export_url(domain, datasource_id):
+    return f"a/{domain}/configurable_reports/data_sources/export/{datasource_id}?format=csv"
+
+def get_datasource_list_url(domain):
+    return f"a/{domain}/api/v0.5/ucr_data_source/"
+
+def get_ucr_database():
+    from superset import db
+    from superset.models.core import Database
+    # Todo; get actual DB once that's implemented
+    return db.session.query(Database).filter_by(database_name="HQ Data").one()
```

Comment:
That should not be the case on a live superset integrated with HQ. I think this should fail hard with a 500 so that we get an alert to fix the case where a DB is not present.